downstream/modules/eda/con-replacing-controller-tokens.adoc

```diff
@@ -0,0 +1,6 @@
+[id="replacing-controller-tokens"]
+
+= Replacing controller tokens in {PlatformName} {PlatformVers}
+
+
+To use {EDAcontroller} in {PlatformName} {PlatformVers}, you must replace legacy controller tokens configured in your environment with {PlatformName} credentials since controller tokens have been deprecated.
```

Comment:
the IBM style guide recommends we don't use "since" as a replacement for "because": https://www.ibm.com/docs/en/ibm-style?topic=word-usage